lib/nexpose/site_credentials.rb

```diff
@@ -67,6 +67,7 @@ def self.for_service(name, id = -1, desc = nil, host = nil, port = nil, service
       cred.port_restriction = port
       cred.service = service
       cred.scope = Credential::Scope::SITE_SPECIFIC
+      cred.permission_elevation_type = ElevationType::NONE
```

Comment:
Make sure to prefix `ElevationType::NONE` with `Credential`. It should resolve fine here, but let's be explicit as possible!